system/Email/Email.php

```diff
@@ -2367,7 +2367,10 @@ public function printDebugger($include = ['headers', 'subject', 'body'])
 
 		is_array($include) || $include = [$include]; // @phpstan-ignore-line
 
-		in_array('headers', $include, true) && $rawData  = htmlspecialchars($this->headerStr) . "\n";
+		if (in_array('headers', $include, true))
+		{
+			$rawData = htmlspecialchars($this->headerStr) . "\n";
+		}
 		in_array('subject', $include, true) && $rawData .= htmlspecialchars($this->subject) . "\n";
 		in_array('body', $include, true) && $rawData    .= htmlspecialchars($this->finalBody);
```

Comment:
If we could have the Rector rule cover these two also that would be great.

Reply:
Yes, hopefully next improvement 👍

Reply:
implemented in rector dev-master, wait for next release ;)